packages/teams-js/prepBetaRelease.js

```diff
@@ -146,32 +139,22 @@ function getNewPkgJsonContent(currBetaVer) {
 
   // get package version from package.json
   let currPkgJsonVer = getPkgJsonVersion(packageJson);
-  let currBetaPrefix = getPrefix(currBetaVer);
 
   console.log('package.json version: ' + currPkgJsonVer);
   console.log('current beta tagged version: ' + currBetaVer);
+  const newVersionPrefix = getNextPrefix(currBetaVer, currPkgJsonVer);
 
-  if (currPkgJsonVer.includes('beta')) {
-    throw new Error(
-      `The given package.json\'s version ${currPkgJsonVer} contains the substring \'beta\' which is reserved for non-prod versions. Please fix the package.json version first in order to allow for proper version incrementation.`,
-    );
-  }
-  let betaVerNum = 0;
-  if (currBetaPrefix === currPkgJsonVer) {
-    betaVerNum = getSpecificVerSuffixNum('beta', currPkgJsonVer) + 1;
-  } else {
-    currBetaPrefix = currPkgJsonVer;
-  }
+  const betaVerNum = getNewBetaSuffixNum(currBetaVer, currPkgJsonVer, newVersionPrefix);
   if (isNaN(betaVerNum)) {
     throw new Error(
       `The given package.json\'s version ${currPkgJsonVer} has a non-integer beta version number. Please fix the package.json version first in order to allow for proper version incrementation.`,
     );
   }
-  const newVersion = currBetaPrefix + '-beta.' + betaVerNum;
+  const newVersion = newVersionPrefix + '-beta.' + betaVerNum;
 
   console.log('new version: ' + newVersion);
 
-  // update package.json with the new version
+  //update package.json with the new version
```

Comment:
nit: keeping a space after the comments is more consistent with the rest of the code.